website/docs/r/iam_user.html.markdown

```diff
@@ -16,6 +16,9 @@ Provides an IAM user.
 resource "aws_iam_user" "lb" {
   name = "loadbalancer"
   path = "/system/"
+  tags {
```

Comment:
Since `tags` is a `schema.TypeMap`, there should be an equals sign here. This will be a requirement in Terraform 0.12 and the change is backwards compatible.

```suggestion
  tags = {
```